roles/kubernetes/master/templates/kubeadm-config.yaml.j2

```diff
@@ -59,7 +59,7 @@ apiServerExtraArgs:
 {% endif %}
   allow-privileged: "true"
 {% for key in kube_kubeadm_apiserver_extra_args %}
-  {{ key }}: {{ kube_kubeadm_apiserver_extra_args[key] }}
+  {{ key }}: {{ kube_kubeadm_apiserver_extra_args[key]|string }}
```

Comment:
I don't think the issue is in jinja2 filtering. I think you should write it instead like this:
```
  {{ key }}: "{{ kube_kubeadm_apiserver_extra_args[key] }}"`

Reply:
Fixed